includes/forms/template.php

```diff
@@ -1687,13 +1688,13 @@ function give_checkout_submit( $form_id ) {
 		 *
 		 * @since 1.7
 		 */
-		do_action( 'give_donation_form_after_submit', $form_id );
+		do_action( 'give_donation_form_after_submit', $form_id, $args );
 		?>
 	</fieldset>
 	<?php
 }
 
-add_action( 'give_donation_form_after_cc_form', 'give_checkout_submit', 9999 );
+add_action( 'give_payment_fields_bottom', 'give_checkout_submit', 9999, 2 );
```

Comment:
@mehul0810 This can cause of breaking UI. Please try to add max priority to this action with existing hook.